src/ui/components/NetworkMonitor/TypesDropdown.tsx

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Are we supposed to be adding this to every new file, or is there some particular criteria for when we do or don't add it?